app/index.js

```diff
@@ -190,14 +192,30 @@ module.exports = yeoman.Base.extend({
     },
 
     composerJson: function () {
-      var composer = this.fs.readJSON('composer.json');
+      // Import any existing composer.json that was saved above.
+      var composer = {};
+      var isNewProject = (this.composerOrig == undefined);
```

Comment:
Since if there is already a composer.json there's already code in the project. That makes sense. I wonder if keying this on the `--replay` parameter, but it occurs to me that always merging into an existing composer.json if it exists is probably a feature. This one might be a good simple check to move into the lib/ directory.